npm-cli/.npmignore

```diff
@@ -1 +1,2 @@
-src/*
\ No newline at end of file
+src/*
+share/*
```

Comment:
Should just remove the share folder probably

Reply:
Yeah. It wouldn't hurt to publish the .re files to npm.

Reply:
Okay, I can just remove this from the ignore again then. They are not used in this PR anymore but I guess they don't hurt either?